--- configure.ac	2019-06-06 19:23:44.000000000 +0300
+++ configure.ac_fix	2019-09-09 23:53:35.000000000 +0300
@@ -1196,7 +1196,7 @@
 
 AC_DEFINE_UNQUOTED(UUID_FILE,"$localstatedir/lib/heartbeat/hb_uuid", Location of Heartbeat's UUID file)
 
-OCF_ROOT_DIR=`try_extract_header_define $GLUE_HEADER OCF_ROOT_DIR /usr/lib/ocf`
+OCF_ROOT_DIR=`try_extract_header_define $GLUE_HEADER OCF_ROOT_DIR $libdir/ocf`
 if test "X$OCF_ROOT_DIR" = X; then
   AC_MSG_ERROR(Could not locate OCF directory)
 fi
